src/app/public/modules/text-editor/menubar/text-editor-menubar.component.ts

```diff
@@ -0,0 +1,190 @@
+import {
+  Component,
+  Input,
+  ViewEncapsulation,
+  OnInit,
+  OnDestroy,
+  ChangeDetectionStrategy
+} from '@angular/core';
+
+import {
+  Subject
+} from 'rxjs';
+
+import {
+  takeUntil
+} from 'rxjs/operators';
+
+import {
+  SkyTextEditorMenubarAction
+} from '../types/menubar-action';
+
+import {
+  SkyTextEditorService
+} from '../services/text-editor.service';
+
+import {
+  SkyTextEditorMergeField
+} from '../types/text-editor-merge-field';
+
+import {
+  SkyTextMergeFieldService
+} from '../services/text-merge-field.service';
+
+import {
+  SkyDropdownMessage,
+  SkyDropdownMessageType
+} from '@skyux/popovers';
+
+/**
+ * @internal
+ */
+@Component({
+  selector: 'sky-text-editor-menubar',
+  templateUrl: './text-editor-menubar.component.html',
+  styleUrls: ['./text-editor-menubar.component.scss'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  encapsulation: ViewEncapsulation.None
+})
+export class SkyTextEditorMenubarComponent implements OnDestroy, OnInit {
+
+  @Input()
+  public editorFocusStream = new Subject();
+
+  @Input()
+  public editorId: string;
+
+  @Input()
+  public menubarActions: SkyTextEditorMenubarAction[] = [];
+
+  @Input()
+  public mergeFields: SkyTextEditorMergeField[] = [];
+
+  public editDropdownStream = new Subject<SkyDropdownMessage>();
+
+  public editItems = [
+    {
+      function: () => this.execCommand('undo'),
+      label: 'Undo',
```

Comment:
All of these labels are not localized. That seems like it could be problematic.

Reply:
Same for the other arrays here.

Reply:
All the menubar strings in the `.ts` file have now been localized.